src/types/common/enums/common.enums.ts

```diff
@@ -35,7 +35,8 @@ export enum VariantEnum {
   Button = 'button',
   H5 = 'h5',
   H6 = 'h6',
-  Body2 = 'body2'
+  Body2 = 'body2',
```

Comment:
if you alredy modify this enum. split it to typographyVariantEnum and buttonVariantEnum

Reply:
i didn't get what do you mean where to split it

Reply:
As I understand you this enum contain variants for buttons and typography, Mickhael say, that would be better to create two enums instead, typographyVariantEnum and buttonVariantEnum